android/src/main/java/com/pylon/RNSpokestack/RNSpokestackModule.java

```diff
@@ -86,6 +93,17 @@ public void initialize(ReadableMap config) throws Exception {
                     .setAndroidContext(reactContext.getApplicationContext())
                     .build();
         }
+
+        // NLU
+        if (config.hasKey("nlu")) {
+            final TensorflowNLU.Builder nluBuilder = new TensorflowNLU.Builder();
+            nluBuilder.setConfig(pipeline.getConfig());
+            nluBuilder.addTraceListener(this);
+            Map<String, Object> map = config.getMap("nlu").toHashMap();
+            for (String k : map.keySet())
+                nluBuilder.setProperty(k, map.get(k));
```

Comment:
Also, only using this method means RN users can't register their own slot parsers.

Reply:
yeah, i think it's a reasonable limitation for the moment, but I'll put in a card to expand the configuration allowed in the future.